time stamp
kineverse/Model update              # Update containing additions and modifications
string[]        deleted_paths       # Paths of deleted data
string[]        deleted_constraints # Identifiers of deleted constraints